packages/g/g4music/package.yml

```diff
@@ -0,0 +1,24 @@
+name       : g4music
+version    : "3.4_1"
+release    : 1
+source     :
+    - https://gitlab.gnome.org/neithern/g4music/-/archive/v3.4-1/g4music-v3.4-1.tar.gz : a3eac9200efddb1aa99c49986edc50e2938f06e81647914bd1be41f8577222d1
+homepage   : https://gitlab.gnome.org/neithern/g4music
+license    : GPL-3.0-or-later
+component  : multimedia.audio
+summary    : Play your music elegantly.
+description: |
+    G4Music is a fast fluent lightweight music player written in GTK4, with a beautiful and adaptive user interface, focuses on high performance for large music collection.
+builddeps  :
+    - pkgconfig(gstreamer-1.0)
```

Comment:
`pkgconfig(gstreamer-1.0)` is a dependency of `pkgconfig(gstreamer-tag-1.0)` so it can be removed.